version_check.py

```
@@ -0,0 +1,35 @@
+        
+        # Получаем базовую версию (без +XX)
+        base_version = BOT_VERSION.split('+')[0]
+        
+        if base_version in changelog:
+            print(f"📝 Changelog для версии {base_version}:")
+            for change in changelog[base_version]["ru"]:
+                print(f"   ✅ {change}")
+        else:
+            print(f"⚠️ Changelog для версии {base_version} не найден")
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)